addnewcar_bp.py

```diff
@@ -0,0 +1,60 @@
+from flask import Blueprint, render_template, request, redirect, url_for, flash, abort
+from flask_login import current_user
+import db_functions
+import email_functions
+
+
+blueprint = Blueprint('addnewcar_bp', __name__)
+@blueprint.route('/noveauto', defaults={'id_zavod':0})
+@blueprint.route('/noveauto/<id_race>', )
+def show_adding_new_car_control(id_race):
+	races = db_functions.races()
+	variable = request.args
+	id_race = variable.get('id_zavod')
+	if not id_race:
+		id_race = 0
+	# print(id_zavod)
+	user = current_user
+
+	if user.is_authenticated:
+		if int(id_race) > 0:
+			result = db_functions.car_exists(user.db_id, id_race)
+			if result:
+				flash ('Na tento závod už auto nabízíš;)', "error")
+				return render_template('newautoinput.html', zavody=races, id_vybraneho=int(id_race), values={}, error=chyba)
```

Comment:
The variable `chyba`  is not defined, you can remove the `error` completely, it's been replaced by `flash`es anyway.